dist/profile/manifests/letsencrypt.pp

```diff
@@ -11,11 +11,4 @@
   package { 'python-certbot-apache':
     ensure => present
   }
-
-  cron { 'letsencrypt-renew-reload':
-    ensure  => present,
-    command => '/opt/letsencrypt/letsencrypt-auto renew --quiet --renew-hook="service apache2 reload"',
```

Comment:
this command doesn't exist on ci.jenkins.io (although still present in the crontab)
I assume left over from a previous version of the puppet module